src/gridcoin/upgrade.cpp

```diff
@@ -571,3 +588,37 @@ void Upgrade::DeleteSnapshot()
         LogPrintf("Snapshot Downloader: Exception occurred while attempting to delete snapshot (%s)", e.code().message());
     }
 }
+
+bool Upgrade::SyncFromZero()
+{
+    return CleanupBlockchainData();
+}
+
+std::string Upgrade::BlockchainCleanupInstructions()
+{
+    std::stringstream stream;
+
+    // Little more work then needed but we should be translation friendly imo
+    stream << _("Datadir: ");
+    stream << GetDataDir().string();
+    stream << "\r\n\r\n";
+    stream << _("Due to the failure to delete the blockchain data you will be required to manually delete the data before starting your wallet.");
+    stream << "\r\n";
+    stream << _("Failure to do so will result in undefined behaviour or failure to start wallet.");
+    stream << "\r\n\r\n";
+    stream << _("You will need to delete the following.");
+    stream << "\r\n\r\n";
+    stream << _("Files:");
+    stream << "\r\n";
+    stream << "blk000*.dat";
```

Comment:
Shouldn't we mention all the other blk***.dat files? (currently there's blk001.dat and blk002.dat, but we should aim to not hardcode it with just mentioning those two)